src/main/java/sirius/biz/storage/layer2/BlobController.java

```diff
@@ -86,4 +85,25 @@ public void uploadFile(final WebContext ctx,
                             .handle();
         }
     }
+
+    @Routed(value = "/dasd/blob-info-for-path/:1", jsonCall = true)
+    @LoginRequired
+    public void blobInfoForPath(final WebContext ctx, JSONStructuredOutput out, String spaceName) {
+        String path = ctx.getParameter("path");
+
+        Blob blob = blobStorage.getSpace(spaceName)
+                               .findByPath(path)
+                               .orElseThrow(() -> Exceptions.createHandled()
+                                                            .withSystemErrorMessage(
+                                                                    "Cannot find blob for path '%s' in space '%s'",
+                                                                    path,
+                                                                    spaceName)
+                                                            .handle());
+
+        out.property("fileId", blob.getBlobKey());
+        out.property("filename", blob.getFilename());
+        out.property("size", blob.getSize());
+        out.property("formattedSize", NLS.formatSize(blob.getSize()));
+        out.property("downloadUrl", blob.url().asDownload().buildURL().orElse(""));
```

Comment:
isn't this a security issue? as "anyone" (who is REALLY GOOD as guessing ids) could obtain a download link - coudln't we handle this on another layer?

Reply:
or maybe this is all good and properly enforced - just a random thought ;-)